braddeibert/musclenerds
# Navigation setup and home screen layout

Description: adds basic navigation structure to the app and home screen view template (still missing certain fragments).

**Changes:**
* Adds navigation options (home, workout catalog, exercise library, track workout, HR monitoring)
* Adds a bottom navigation bar that mirrors the functionality in navigation drawer
* Changes the application theme to black/gray/white - no more planet fitness purple!!!
* Adds global string values to app in string.xml

**Testing:**
* Use navigation drawer to navigate around the app. The title in the top bar should update as you change views.
* Use bottom navigation bar to navigate around the app. The title in the top bar should update as you change views.
* Be sure that the app now contains a black/gray/white color scheme.
